packages/tools/devtools/devtools-browser-extension/jest.config.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Get the test port from the global map and set it in env for this test
+const testTools = require("@fluidframework/test-tools");
+const { name } = require("./package.json");
+
+mappedPort = testTools.getTestPort(name);
+process.env["PORT"] = mappedPort;
+
+module.exports = {
+	preset: "jest-puppeteer",
+	globals: {
+		PATH: `http://localhost:${mappedPort}`,
+	},
+	testTimeout: 30000,
```

Comment:
Nit: This is really long. Do we need this to be so long, or could we just use the default?

Reply:
Removed. It will use default timeout as 5000

Reply:
Seems like 5000 will cause timeout sometimes. Revoke back